### PR TITLE
Added function "hightling_data"

### PR DESCRIPTION
**Pull request by JABillard**
Added a function, which given a dataframe, will highlight the paths accordingly. The columns must be the name of functions used by the fram model. Values must be 0 (absent) or 1 (present) for each. Paths will be highlighted based on the number of times both functions of a single curve have a value of 1 over all instances.

### CODE REVIEW

File: framalytics/fram.py [hunk 1]
```diff
@@ -216,6 +217,71 @@ def generate_connection_traversal_count(self, output_function):
...
+            # This is based on if both functions in an instance have a value of 1.
+            for row in range(len(data[outputFn_name])):
+                outputFn_data_value = data[outputFn_name][row]
+                toFn_data_value = data[toFn_name][row]
+
+                if ((outputFn_data_value == 1) and (toFn_data_value == 1)):
+                    connection_value = self.connections.get(connection_default)
+                    connection_value += 1
+                    self.connections.update({connection_default: connection_value})
+
```
Comment (ttricco) [inline framalytics/fram.py:207]: We can avoid this inner loop by using pandas slicing and len().

`connection_value = len(data[(data[outputFn_name] == 1) & (data[toFn_name] == 1)])`

File: framalytics/FRAM_Visualizer.py [hunk 2]
```diff
@@ -328,7 +328,8 @@ def generate_full_path_from_function(self, aspect_data, output_function):
 
             already_pathed.append(current_function)
 
-    def bezier_curve_single(self,curve):
+
+    def bezier_curve_single(self,curve,color = None):
         """
         :param curve: The curve string between two aspects which is given by the parser.
```
Comment (ttricco) [inline framalytics/FRAM_Visualizer.py:352]: Add color as a param in the docstring.

**Review by ttricco — CHANGES_REQUESTED** [review]

**Comment by JABillard** [comment]
Updated Pull Request to include the most recent changes. This included updated documentation and minor error fixes.

**Review by JABillard — COMMENTED** [review]
Added fnStyles to the figure, which alter the location of aspects based on a functions style. Currently all functions will share the style of the first function in the .xfmv file.

**Review by ttricco — APPROVED** [review]

**Pull request merged**